### PR TITLE
Fix Handling Quit Requests wording about mobile platforms

### PR DESCRIPTION
**Pull request by ztc0611**
Closes https://github.com/godotengine/godot-docs/issues/8474.

Relies on https://github.com/godotengine/godot/pull/85100.

### CODE REVIEW

File: tutorials/inputs/handling_quit_requests.rst [hunk 1]
```diff
@@ -62,6 +49,22 @@ procedure:
 
     GetTree().AutoAcceptQuit = false;
 
+On Mobile Devices
```
Comment (AThousandShips) [inline tutorials/inputs/handling_quit_requests.rst:52]:
```suggestion
On mobile devices
```

File: tutorials/inputs/handling_quit_requests.rst [hunk 2]
```diff
@@ -62,6 +49,22 @@ procedure:
...
+There is no direct equivalent of ``NOTIFICATION_WM_CLOSE_REQUEST`` on mobile 
+platforms. Due to the nature of mobile operating systems, the only place 
+that you can run code prior to quitting is when the app is being suspended to 
+the background. On both Android and iOS, the app can be killed while suspended 
+at any time by either the user or the OS. A way to plan ahead for this 
+possibility is to utilize ``NOTIFICATION_APPLICATION_PAUSED`` in order to 
+perform any needed code as the app is suspended.
+
+On Android, pressing the Back button will exit the application if 
+Application > Config > Quit On Go Back is checked in the Project Settings
```
Comment (AThousandShips) [inline tutorials/inputs/handling_quit_requests.rst:64]:
```suggestion
**Application > Config > Quit** On Go Back is checked in the Project Settings 
```
Don't remove this

File: tutorials/inputs/handling_quit_requests.rst [hunk 3]
```diff
@@ -62,6 +49,22 @@ procedure:
 
     GetTree().AutoAcceptQuit = false;
 
+On Mobile Devices
+-----------------
+
+There is no direct equivalent of ``NOTIFICATION_WM_CLOSE_REQUEST`` on mobile
```
Comment (AThousandShips) [inline tutorials/inputs/handling_quit_requests.rst:55]:
```suggestion
There is no direct equivalent to ``NOTIFICATION_WM_CLOSE_REQUEST`` on mobile 
```

File: tutorials/inputs/handling_quit_requests.rst [hunk 4]
```diff
@@ -8,8 +8,8 @@ Quitting
 
 Most platforms have the option to request the application to quit. On
 desktops, this is usually done with the "x" icon on the window title bar.
-On Android, the back button is used to quit when on the main screen (and
-to go back otherwise).
+On mobile devices, the app can be quit at any time while it is suspended
```
Comment (AThousandShips) [inline tutorials/inputs/handling_quit_requests.rst:11]:
```suggestion
On mobile devices, the app can quit at any time while it is suspended
```

File: tutorials/inputs/handling_quit_requests.rst [hunk 5]
```diff
@@ -62,6 +49,22 @@ procedure:
...
+On Mobile Devices
+-----------------
+
+There is no direct equivalent of ``NOTIFICATION_WM_CLOSE_REQUEST`` on mobile 
+platforms. Due to the nature of mobile operating systems, the only place 
+that you can run code prior to quitting is when the app is being suspended to 
+the background. On both Android and iOS, the app can be killed while suspended 
+at any time by either the user or the OS. A way to plan ahead for this 
+possibility is to utilize ``NOTIFICATION_APPLICATION_PAUSED`` in order to 
+perform any needed code as the app is suspended.
```
Comment (AThousandShips) [inline tutorials/inputs/handling_quit_requests.rst:61]:
```suggestion
perform any needed actions as the app is being suspended.
```

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you! Merged.

**Pull request merged**